src/services/jobPosts/setJobPostsAPI.ts

```diff
@@ -91,6 +93,18 @@ const jobPostsAPI = createApi({
         },
       }),
     }),
+    filterJobPosts: builder.query<
+      { data: IWorkCardProps[]; totalCount: number },
```

Comment:
Move to interface. Watch on code below and make in same style, it's mach more easier to work with api query params type and api's return values type when their types declared